client.go

```diff
@@ -480,3 +456,83 @@ ExitForLoop:
 		}
 	}
 }
+
+func runHTTPLatencyTest(test *ethrTest) {
+	uri := test.session.remoteAddr
+	ui.printMsg("uri=%s", uri)
+	uri = "http://" + uri + ":" + httpLatencyPort
+
+	buff := make([]byte, test.testParam.BufferSize)
+	for i := uint32(0); i < test.testParam.BufferSize; i++ {
+		buff[i] = 'x'
+	}
+	
+	rttCount := test.testParam.RttCount
+	latencyNumbers := make([]time.Duration, rttCount)
+	tr := &http.Transport{DisableCompression: true}
+	client := &http.Client{Transport: tr}
+ExitForLoop:
+	for {
+	ExitSelect:
+		select {
+		case <-test.done:
+			break ExitForLoop
+		default:
+			for i := uint32(0); i < rttCount; i++ {
+				s1 := time.Now()
+				response, err := client.Post(uri, "text/plain", bytes.NewBuffer(buff))
+				if err != nil {
+					break ExitSelect
+				} else {
+					if response.StatusCode != http.StatusOK {
+						break ExitSelect
+					}
+					contents, err := ioutil.ReadAll(response.Body)
+					response.Body.Close()
+					if err != nil {
+						break ExitSelect	
+					}
+					ethrUnused(contents)
+				}
+				
+				e2 := time.Since(s1)
+				latencyNumbers[i] = e2
+			}
+
```

Comment:
Pls remove extra spacing. Also run go fmt.

Reply:
Can you tell what do you mean by go fmt?